src/components/AssociatedGenes.js

```diff
@@ -47,24 +136,88 @@ const getDataAll = genesForVariant => {
   return data;
 };
 
-const getTissueColumns = (schemas, sourceId) => {
-  const columns = [{ id: 'gene', label: 'Gene' }];
-  schemas
-    .find(schema => schema.sourceId === sourceId)
-    .tissues.forEach(tissue => {
-      columns.push({
-        id: tissue.id,
-        label: tissue.name,
-        renderCell: rowData => {
-          return rowData[tissue.id]
-            ? rowData[tissue.id].toPrecision(3)
-            : 'No information';
-        },
-      });
+const getTissueColumns = (
+  genesForVariantSchema,
+  genesForVariant,
+  sourceId,
+  classes
+) => {
+  const columns = [{ id: 'geneSymbol', label: 'Gene' }];
+  let type;
+  let schema;
+
+  genesForVariantSchema.qtls.forEach(s => {
```

Comment:
This is fine; just a stylistic comment. I found it quite hard to follow the setting of `type` and `schema`. You could do something like the following instead?
```
const schemaForTab = [
  ...genesForVariantSchema.qtls.map(d => ({ ...d, type: 'qtls' })),
  ...genesForVariantSchema.intervals.map(d => ({ ...d, type: 'intervals' })),
  ...genesForVariantSchema.functionalPredictions.map(d => ({ ...d, type: 'functionalPredictions' }))
].find(d => d.sourceId === sourceId);

let tissueColumns;
switch (schemaForTab.type) {
  case 'eqtls':
    tissueColumns = schemaForTab.tissues.map(...)
  case 'intervalss':
    tissueColumns = schemaForTab.tissues.map(...)
  case 'functionalPredictions':
    tissueColumns = schemaForTab.tissues.map(...)
}

const columns = [
  { id: 'geneSymbol', label: 'Gene' },
  ...tissueColumns
];
```